src/lib/components/ui/Button/Button.jsx

```diff
@@ -5,48 +5,79 @@ import withForwardedRef from '../withForwardedRef';
 import styles from './Button.scss';
 
 export const Button = (props) => {
```

Comment:
I would write it as:
```
export const Button = ({
    afterLabel,
    beforeLabel,
    block,
    clickHandler,
    disabled,
    endCorner,
    forwardedRef,
    grouped,
    id,
    label,
    labelVisibility,
    loadingIcon,
    priority,
    size,
    startCorner,
    type,
    variant,
  }) => {
```

Reply:
I see, but… really? That's yet another approach, not yet used anywhere in the library…

Reply:
I think the suggested approach is more concise. I don't think it matters, that it it not used anywhere else - its not a big thing, just syntactic sugar.

Reply:
It is not possible due to `transferProps` functionality.